hollow/src/main/java/com/netflix/hollow/api/producer/HollowProducer.java

```diff
@@ -630,15 +640,18 @@ private void applyDelta(Blob blob, HollowReadStateEngine stateEngine) throws IOE
 
     private void validate(HollowProducer.ReadState readState) {
         ProducerStatus.Builder status = listeners.fireValidationStart(readState);
+        ValidationStatus.Builder valStatus = ValidationStatus.builder().withReadState(readState).withVersion(readState.getVersion());
         try {
             List<Throwable> validationFailures = new ArrayList<Throwable>();
             
             for(Validator validator : validators) {
+            	Throwable throwable = null;
```

Comment:
Wonder whether it might be better to have validate return a result instead of throwing exception.  Something to consider for the future.

Reply:
Yes, that approach is being considered for next iteration.